Dockerfile

```
@@ -5,17 +5,20 @@ RUN apt-get update \
   && apt-get install -y git \
```

```suggestion
  && apt-get install -y \
    git \
```

putting the packages on newlines makes sorting/managing them later easier

---

Also, why do you need `git` in this image?